scripts/apps/authoring/authoring/directives/SendItem.js

```diff
@@ -540,6 +552,28 @@ export function SendItem($q, api, desks, notify, authoringWorkspace,
                 return $q.when(item);
             }
 
+            const confirmMovePackage = () =>
+                modal.confirm(
+                    gettext('On current package there are already published items that can\'t be moved.'),
```

Comment:
Grammar fix: The package contains published items which can\'t be moved.

Reply:
Done

Reply:
Thanks. Can you address the remaining comment? :)